//
//  Copyright (C) 2017 Skyloupe, LLC
// 
//  Licensed under the Apache License, Version 2.0 (the "License").
//  You may not use this file except in compliance with the License.
//  A copy of the License is located at
//  
//  https://www.apache.org/licenses/LICENSE-2.0.html
//  
//  or in the "license" file accompanying this file. This file is distributed
//  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
//  express or implied. See the License for the specific language governing
//  permissions and limitations under the License.
//

syntax = "proto3";

package skyloupe.skytrans;

option go_package = "com.skyloupe.skytrans";

message Stencil
{
    map<string, string> string_values = 100;
    map<string, int32> int_values     = 200;
    map<string, bool> bool_values     = 300;

    map<string, uint32> uint_values   = 400;
    map<string, int64> int64_values   = 500;
    map<string, uint64> uint64_values = 600;
}
